modules/ROOT/pages/configure-ingress-http-resource.adoc

```diff
@@ -0,0 +1,228 @@
+
+* Wildcard for host-based routing.
+* Creation of `httproutetemplates.rtf.mulesoft.com` templates to map a set of application endpoints to arbitrary HTTP routing resources for path-based routing..
+* Synchronization of domains or hosts from the HTTP route templates with the Anypoint control plane.
+* Using parameter placeholders in `httproutetemplates.rtf.mulesoft.com` templates for additional extensibility.
+
+== HTTP Route-Template Custom Resource Placeholders
+
+Runtime Fabric reveals placeholder properties that cluster administrators can use in their HTTP route-template custom resource. These placeholders are evaluated and replaced with actual data by Runtime Fabric when rendering the deployment charts.
+
+As a cluster administrator, use these HTTP route-template placeholder properties in your CRD:
+
+[%header%autowidth.spread]
+|===
+| Property| Description
+| `{{ .ResourceName }}`| Replaces with the Runtime Fabric generated unique routing.
+| `{{ .ApplicationName }}`| Replaces with the Mule app deployment name.
+| `{{ .Namespace }}`| Replaces with the namespace of the Mule app deployment.
+| `{{ .Service.Name }}`| Replaces with the backend service name of the Mule app deployment.
+| `{{ .Service.PortName }}`| Replaces with the backend service port name of the Mule app deployment.
+| `{{ .Service.Port }}`| Replaces with the backend service port number of the Mule app deployment.
+| `{{ .Host }}`| Replaces with the host or domain value from the public endpoint of Mule  app deployment.
+| `{{ .Path }}`| Replaces with the path value from the public endpoint of Mule app deployment.
+| `{{ .OrganizationID }}`| Replaces with the organization ID of the Mule app deployment.
+| `{{ .EnvironmentID }}`| Replaces with the environment ID of the Mule app deployment.
+| `{{ .UniqueId }}`| Replaces with the unique ID.
+|===
+
+== How HTTP Route-Template Custom Resource Works in Runtime Fabric
+
+This diagram provides an overview of how to configure ingress using an HTTP route-template custom resource in Runtime Fabric:
+
+image::ingress-http-diagram.png[HTTP route-template diagram overview]
+
+The `httproutetemplates` resource applies only to new Mule app deployments created after the template resource is set up. It does not affect existing deployments. If you redeploy a Mule app after creating or updating the `httproutetemplates`, the new template is applied.
+
+The `httproutetemplates` resource enables you to configure routes like these:
+
+* `baseEndpoints` +
+Describes a list of endpoints associated with the routing resources defined in the template. These endpoints are synchronized by Runtime Fabric with the Anypoint control plane. Application developers can select from a list of synchronized base endpoints during deployment and optionally customize the path and subdomain when applicable.
```

Comment:
Is this correct: "Anypoint control plane"?

Reply:
Can you get rid of "Describes a list of" and just start the sentence with "Endpoints"?